### PR TITLE
Support identifiers as context decls in human-readable schemas

### PR DESCRIPTION
**Pull request by shaobo-he-aws**
## Checklist for requesting a review

The change in this PR is (choose one, and delete the other options):

- [x] A backwards-compatible change requiring a minor version bump to `cedar-policy` (e.g., addition of a new API).

I confirm that this PR (choose one, and delete the other options):

- [x] Updates the "Unreleased" section of the CHANGELOG with a description of my change (required for major/minor version bumps).

I confirm that [`cedar-spec`](https://github.com/cedar-policy/cedar-spec) (choose one, and delete the other options):

- [x] Does not require updates because my change does not impact the Cedar formal model or DRT infrastructure.

### CODE REVIEW

**Comment by john-h-kastner-aws** [comment]
Is this fixing #681?

**Comment by shaobo-he-aws** [comment]
> Is this fixing #681?

Yes, thanks for pointing it out.

**Review by aaronjeline — APPROVED** [review]
LGTM

File: cedar-policy-validator/src/human_schema/test.rs [hunk 1]
```diff
@@ -317,6 +317,46 @@ mod demo_tests {
         assert!(as_src.contains(expected), "src was:\n`{as_src}`");
     }
 
+    #[test]
+    fn context_is_common_type() {
+        assert!(SchemaFragment::from_str_natural(
```
Comment (khieta) [inline cedar-policy-validator/src/human_schema/test.rs:322]: Could you add some non-empty contexts too for completeness? E.g., the `authcontext` example from the [original RFC](https://github.com/cedar-policy/rfcs/blob/main/text/0024-schema-syntax.md).

**Review by khieta — APPROVED** [review]
Lgtm. Left one small comment about test cases.

**Comment by shaobo-he-aws** [comment]
> Do we need to make the same change for entity type attributes?

Yes, will do it in another PR.

File: cedar-policy-validator/src/human_schema/test.rs (hunk 1 above)
Comment (shaobo-he-aws) [reply cedar-policy-validator/src/human_schema/test.rs:322]: Done.

**Review by aaronjeline — APPROVED** [review]

**Review by aaronjeline — APPROVED** [review]

**Pull request merged**